index.js

```diff
@@ -4,16 +4,23 @@ const { WebClient } = require('@slack/web-api');
 const flatten = require('flat');
 const axios = require('axios');
 
+const SLACK_WEBHOOK_TYPES = {
+    WORKFLOW_BUILDER: 'BUILDER',
+    WORKFLOW_INCOMING: 'INCOMING'
+}
+
 try {
     const botToken = process.env.SLACK_BOT_TOKEN;
     const webhookUrl = process.env.SLACK_WEBHOOK_URL;
+    const webhookType = process.env.SLACK_WEBHOOK_TYPE?.toUpperCase() || SLACK_WEBHOOK_TYPES.WORKFLOW_BUILDER;
```

Comment:
```suggestion
    // The default type is for Workflow Builder triggers. If you want to use this action for Incoming Webhooks, use the corresponding type instead.
    const webhookType = process.env.SLACK_WEBHOOK_TYPE?.toUpperCase() || SLACK_WEBHOOK_TYPES. WORKFLOW_TRIGGER;
```

Reply:
Note: whitespace in variable: `SLACK_WEBHOOK_TYPES. WORKFLOW_TRIGGER`

Reply:
Thanks, fixed.